crates/net/discv4/src/lib.rs

```diff
@@ -941,10 +1187,14 @@ impl Discv4Service {
     }
 
     fn find_node_timeout(&self) -> u64 {
-        (SystemTime::now().duration_since(UNIX_EPOCH).unwrap() + self.config.find_node_timeout)
+        (SystemTime::now().duration_since(UNIX_EPOCH).unwrap() + self.config.request_timeout)
```

Comment:
Given the find_node timeouts have been unified, do we still need the neighbors & other timeouts?